common/authorization/authorizer.go

```diff
@@ -72,7 +74,7 @@ type Authorizer interface {
 
 // @@@SNIPEND
 
-type requestWithNamespace interface {
+type hasNamespace interface {
```

Comment:
the previous name seems better?

Reply:
Agree, I would leave it as it is for now though.